src/containers/CodesplainAppBar.jsx

```diff
@@ -86,8 +86,8 @@ const orderSnippets = (username, userSnippets, orgSnippets) => {
   const allSnippets = { ...orgSnippets }; // Spread so there aren't side effects.
   allSnippets[username] = userSnippets;
   const orderedSnippets = reduce(allSnippets, (allList, snippets, role) => {
-    const next = reduce(snippets, (snipList, { language, lastEdited }, snippetTitle) => {
-      const obj = { role, snippetTitle, language, lastEdited };
+    const next = reduce(snippets, (snipList, { language, lastEdited, snippetTitle }, key) => {
+      const obj = { role, snippetTitle, language, lastEdited, key };
```

Comment:
The line immediately above it
https://github.com/maryvilledev/codesplainUI/blob/15519e6f8e529c556f0b79f9ff97c6314de1a0b5/src/containers/CodesplainAppBar.jsx#L81-L84